internals/self_certify.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2025 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from chromestatus_openapi.models import SurveyAnswers as OASurveyAnswers
+
+from internals.core_enums import GATE_PRIVACY_ORIGIN_TRIAL, GATE_PRIVACY_SHIP
+from internals.review_models import Gate, SurveyAnswers
+
+
+def update_survey_answers(gate: Gate, new_answers: OASurveyAnswers):
```

Comment:
Optional nit: This file could be named `self_certify_helpers.py` since it contains no classes and consists only of helper functions

Reply:
Hmm, it would seem odd to have an X_helpers file without having the X file first.  I'll stick with the shorter name for now, but keep _helpers in mind.